dhctl/pkg/system/ssh/frontend/upload-script.go

```diff
@@ -71,17 +84,35 @@ func (u *UploadScript) Execute() (stdout []byte, err error) {
 	}
 
 	var cmd *Command
+	var scriptFullPath string
 	if u.sudo {
-		cmd = NewCommand(u.Session, "/tmp/"+scriptName, u.Args...).Sudo()
+		scriptFullPath = "/tmp/" + scriptName
+		cmd = NewCommand(u.Session, scriptFullPath, u.Args...).Sudo()
 	} else {
-		cmd = NewCommand(u.Session, "./"+scriptName, u.Args...).Cmd()
+		scriptFullPath = "./" + scriptName
+		cmd = NewCommand(u.Session, scriptFullPath, u.Args...).Cmd()
+	}
+
+	if u.chmod {
```

Comment:
It doesn't look like modifying permission is dhctl job. Our soft should either set permissions automatically or forget about that idea. WDHY? Is there a reason behind adding chmod?

Reply:
Ok. Now we check permissions of script before execution

Reply:
done